docs/getting-started/cli/init.md

```diff
@@ -0,0 +1,30 @@
+---
+description: Initialising new Subsembly project
+---
+
+# subsembly init
+
+### Overview
```

Comment:
I think that we should add information on what is the purpose of init. F.e

"In order for you to start working on a Subsembly based Runtime, you need to initialise new project that will provide all of the necessary boilerplate code with high-level abstractions so that you can focus on the business requirements of your Runtime"